lua/lush_theme/bluloco.lua

```diff
@@ -422,6 +422,55 @@ local theme = lush(function(injected_functions)
     -- BufferLineInfoSelected { fg = t.info, sp = t.primary, gui = "underline" },
     -- BufferLineHintSelected { fg = t.hint, sp = t.primary, gui = "underline" },
 
+    -- BarBar
+
+    BufferCurrent { fg = t.fg, bg = t.bg },
+    BufferCurrentIndex { BufferCurrent },
+    BufferCurrentIcon { BufferCurrentIndex },
+    BufferCurrentMod { BufferCurrent },
+    BufferCurrentSign { fg = t.keyword, bg = BufferCurrent.bg },
+    BufferCurrentTarget { fg = shade(t.error, 40), bg = BufferCurrent.bg },
```

Comment:
Let's use the red from types, and stay consistent for all buffers so it's easier for the eyes? I have added suggestsions for those down below also.
```suggestion
    BufferCurrentTarget { BufferCurrent, fg = t.type },
```